src/sdk/SessionKeyValidator/SessionKeyValidator.ts

```diff
@@ -173,6 +174,29 @@ export class SessionKeyValidator {
         }
     }
 
+    async signUserOpHash(userOpHash: string, userOp: UserOperation): Promise<UserOperation> {
```

Comment:
Refactor and rename this as `signUserOpWithSessionKey`, There can be multiple active session keys per `apiKey` and `walletAddress`. This scenario doesn't handle it.

Reply:
Changes done.